rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/rascal/RascalLanguageServices.java

```diff
@@ -215,7 +215,7 @@ public InterruptibleFuture<ITuple> getRename(ITree cursorTree, Set<ISourceLocati
             } catch (Throw e) {
                 if (e.getException() instanceof IConstructor) {
                     var exception = (IConstructor)e.getException();
-                    if (exception.getType().getAbstractDataType().getName().equals("RenameException")) {
+                    if (exception.getName().contains("Rename")) {
```

Comment:
Is there anyway that now or in the future user code is called that could throw an exception? If so, is "having Rename" in the name discerning enough? Maybe we can also check the type including the "package" part to make sure it is an "official" Rename exception?

Reply:
I agree that this check is not very accurate. For now, however, I do not see a way to check for the origin of the exception.

Reply:
Reverted